spec/granite_orm/fields/casting_spec.cr

```diff
@@ -0,0 +1,25 @@
+# class Review{{ adapter_const_suffix }} < Granite::ORM::Base
```

Comment:
Should we keep this comment?

Reply:
Agree we should remove this. The code tells the story very well on whats going on.

Reply:
I found it difficult to find the model definition so found this useful, but I will remove it.